### PR TITLE
[PT FE] Fix conversion for aten::quantize_per_channel

### PR DESCRIPTION
**Pull request by PiotrKrzem**
### Details:
 - Fix conversion and optimize computations for aten::quantize_per_channel

### Tickets:
 - 115586

### CODE REVIEW

File: tests/layer_tests/pytorch_tests/test_quantize.py [hunk 1]
```diff
@@ -58,30 +58,38 @@ class TestQuantizePerChannelDequantize(PytorchLayerTest):
...
-        0, 1, 2, 3
-    ])
     @pytest.mark.nightly
-    # @pytest.mark.precommit - conversion issue
-    def test_quantize_per_channel_dequantize(self, scales, zero_points, dtype, axis, ie_device, precision, ir_version):
-        if dtype == torch.quint8: zero_points = abs(zero_points)
-        self._test(aten_quantize_per_channel_aten_dequantize(scales, zero_points, dtype, axis), None, ["aten::quantize_per_channel", "aten::dequantize"], 
+    # @pytest.mark.precommit - sporadic issue
+    def test_quantize_per_channel_dequantize(self, scale, zero_point, axis, dtype, ie_device, precision, ir_version):
+        np.random.shuffle(scale), np.random.shuffle(zero_point)
```
Comment (mvafin) [inline tests/layer_tests/pytorch_tests/test_quantize.py:92]: Why is this needed?

File: tests/layer_tests/pytorch_tests/test_quantize.py [hunk 2]
```diff
@@ -58,30 +58,38 @@ class TestQuantizePerChannelDequantize(PytorchLayerTest):
...
+        ],
+        [
+            np.array([1.0, 0.62, 0.74, 0.11, 0.89, 0.32], dtype=np.float32),
+            np.array([0, 2, 4, -5, 6, -7], dtype=np.int32),
+            1
+        ],
+        [
+            np.array([1.0, 0.21, 0.62, 0.5, 0.11, 0.89, 0.32], dtype=np.float32),
+            np.array([0, -1, 2, -3, 4, -5, -7], dtype=np.int32),
+            2
```
Comment (mvafin) [inline tests/layer_tests/pytorch_tests/test_quantize.py:75]: Does it work on CPU?

Comment (PiotrKrzem) [reply tests/layer_tests/pytorch_tests/test_quantize.py]: No, marked with xfail. It was added a few commits back, the merge must've removed it for one second, but now it's back!

**Review by mvafin — APPROVED** [review]
